Makefile

```diff
@@ -51,14 +51,14 @@ _prepare-mobile: ##@prepare Install mobile platform dependencies and prepare wor
 	npm install
 
 $(STATUS_GO_IOS_ARCH):
-	cd $(RCTSTATUS_DIR) && curl -OL "$(GITHUB_URL)/$(STATUS_GO_VER)/status-go-ios.zip"
+	cd $(RCTSTATUS_DIR) && curl -o status-go-ios-$(STATUS_GO_VER).zip -OL "$(GITHUB_URL)/$(STATUS_GO_VER)/status-go-ios.zip"
```

Comment:
can you put this change in a separate commit it is unrelated

Reply:
In a separate PR?

Reply:
No it can be in the same just don't squash them

Reply:
But yes I'll pull it out.